src/Rules/In.php

```diff
@@ -44,6 +44,16 @@ public function check(mixed $value): bool
     {
         $this->assertHasRequiredParameters(['allowed_values']);
 
+        if (is_array($value)) {
```

Comment:
There is "AnyOf" that allows for validating an array of values are each within an array of allowed values. I am not sure how I feel about combining "In" to do both. I checked Laravels code and see that it does allow this, though "AnyOf" allows a string of delineated values as well.